utils.js

```diff
@@ -61,12 +63,15 @@ function getBinaryPath () {
   // run async function synchronously because testem doesn't like async configs
   // I'm not proud of this but it works.
   const versionBuffer = childProcess.execSync(`
-    node -e "
+    CHROMIUM_CHANNEL=${REQUESTED_CHANNEL} CHROMIUM_VERSION=${REQUESTED_VERSION} node -e "
       require('ember-chromium/utils').getExactChromeVersionNumber()
       .then(v => process.stdout.write(v))
     "
   `);
   const versionNumber = String.fromCharCode.apply(null, versionBuffer);
+  if (!versionNumber) {
+    throw new Error(`Failed to locate an official chromium release with major version matching ${REQUESTED_VERSION} in the ${REQUESTED_CHANNEL} channel`);
```

Comment:
This is the second part of the fix.  If the channel/version combo is not found, there is no failure.  Rather, the path will be build without the version.  In the case of a DL (on pkg install or manual invocation), it will just download the latest version of chromium to a non-versioned path of `.../bin/ember-chromium/chrome-{os}/...`.

This bit me when I attempted to use mismatched versions and channels (e.g. a version only in dev and not in beta/stable).

You could argue this fix is a breaking change to the getBinaryPath method, but, seeing that it could DL the version to the wrong location, you can get some split-brained behavior.

I'll leave it to y'all if you think it should be a major rev.